tests/Zilliqa/test_zilliqa_NAT.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
```

Comment:
Apart from the name of the file and the mention of `NAT` in one of the arguments, I can't find any other NAT reference. This likely means that it's a slight modification of another script. Maybe the other script should be (slightly) extended instead of a duplication here. Otherwise, let's do the NAT-related testing functionality more explicit.

Reply:
Created issue #352 to track as it will impact other internal scripts.